test/TestProjects/TypeSchemaMapping/TypeSchemaMapping.json

```diff
@@ -282,6 +282,26 @@
           }
         }
       }
+    },
+    "ModelWithCustomUsage": {
+      "type": "object",
+      "properties": {
+        "ModelProperty": {
+          "type": "string",
+          "description": "."
+        }
+      },
+      "x-csharp-usage": "model,input,output",
```

Comment:
So, would we need to add this in a swagger transform for each of our generated models in Event Grid?

/cc @kerri-lee

Reply:
For you it's `output, model`

Reply:
And `x-csharp-formats : json`